src/reporting/Interfaces.sol

```diff
@@ -9,12 +9,15 @@ import 'ROOT/reporting/ReportingToken.sol';
 import 'ROOT/reporting/ReportingWindow.sol';
 import 'ROOT/reporting/DisputeBondToken.sol';
 import 'ROOT/reporting/RegistrationToken.sol';
-
+import 'ROOT/trading/Cash.sol';
```

Comment:
This shouldn't be imported here, instead it should be imported into whichever contracts need `Cash`.  As we migrate more and more code to Solidity, this file should shrink until it is deleted.